### DISCUSSION TITLE
TestSuite für automatisierte Tests

### DISCUSSION THREAD

**Discussion by Brovning** [elueckel/Unifi-Toolbox#36] [Ideas]
Ähnlich der Action für den CodeStyle-Check kann bei jedem Checkin auch automatisch eine TestSuite als Action ausgeführt werden, um die grundlegende Funktionalität automatisch zu testen und sicherzustellen (siehe: https://youtu.be/kTcuTO50Jzw).

Für jeden gemeldeten Fehler sollte dann zukünftig auch ein Testfall zum abtesten erstellt werden.

Das Problem, was ich aktuell sehe, dass man einen curl-Stub selbst erstellen müsste...

**Comment by Brovning** [comment]
ticket created: #53 
--> discussion locked

**Comment by elueckel** [comment]
Mit dem neuen Modul schlagen die Tests fehl ... willst du die Tests anpassen oder wollen wir die entfernen? Ich bin nicht sicher wieviel Mehrwert die Tests liefern, kann sie aber auch nicht anpassen/erstellen für das neue Modul. Was denkst du? @Brovning

**Reply by Brovning** [reply]
Hallo,

schaue ich mir nachmittags/abends an.

**Reply by Brovning** [reply]
done
